src/schema/uuid/thread/types.ts

```diff
@@ -60,7 +73,29 @@ export interface ThreadResolvers {
     author: Resolver<CommentPayload, never, UserPayload>
   } & UuidResolvers
   Mutation: {
-    createThread: MutationResolver<MutationCreateThreadArgs, ThreadData | null>
+    thread: MutationNamespace
+  }
+  ThreadMutation: {
+    createThread: MutationResolver<
+      ThreadMutationCreateThreadArgs,
+      MutationResponseWithRecord<ThreadData | null>
+    >
+    createComment: MutationResolver<
+      ThreadMutationCreateCommentArgs,
+      ThreadCreateCommentResponse
```

Comment:
```suggestion
      MutationResponse
```

Then you do not need the `... ? ... : ...` magic in the type `MutationResolver`

Reply:
Good idea.
Okay if we keep it until we are a bit more that the Response types won't change so soon?

Reply:
fine for me.